crypto/src/lib.rs

```diff
@@ -182,6 +196,28 @@ pub trait Crypto {
         info: &[u8],
     ) -> Result<Self::Cdi, CryptoError>;
 
+    /// CFI wrapper around derive_cdi_exported
+    ///
+    /// To implement this function, you need to add the
+    /// cfi_impl_fn proc_macro to derive_exported_cdi.
+    #[cfg(not(feature = "no-cfi"))]
+    fn __cfi_derive_exported_cdi(
+        &mut self,
+        algs: AlgLen,
+        measurement: &Digest,
+        info: &[u8],
+    ) -> Result<Self::Cdi, CryptoError>;
+
+    /// Retrieves the CDI for an exported private key.
+    fn get_exported_cdi(&mut self) -> Result<Self::Cdi, CryptoError>;
```

Comment:
I'm not sure this will be necessary after moving `SignWithExported` from DPE to "caliptra-sw".

Leaving it in for now.

Reply:
Agreed I don't think this is needed

Reply:
I've removed this.